react/utils/useCssHandles.tsx

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from 'react'
+import { useOwnerBlock } from '../components/OwnerBlock'
+
+/**
+ * Useful for creating CSS handles without creating a CSS file with empty
+ * declarations.
+ * Receives an array of strings ('foo', 'bar') and returns an object with
+ * { foo: 'vendor-appname-1-x-foo', bar: 'vendor-appname-1-x-bar' }.
+ */
+export const useOwnerBlockCssHandles = (handles: string[] = []): CssHandles => {
+  const { identifier } = useOwnerBlock()
+
+  const values = useMemo(() => {
+    if (!identifier) {
+      const fallbackValues = useMemo(() => {
```

Comment:
you shouldn't call a hook inside a condition. either remove the memo or move it up in the scope